boards/arm/b_u585i_iot02a/b_u585i_iot02a-common.dtsi

```diff
@@ -159,15 +159,41 @@
 		data-rate = <OSPI_DTR_TRANSFER>;
 		four-byte-opcodes;
 		status = "okay";
+		sfdp-bfp = [
```

Comment:
The sfdp-bfp is added as a property of the mx25lm51245: ospi-nor-flash for the stm32l562e_dk board
since it can not be read from the nor octoflash  : "ospi-nor-flash@0: device not ready"
This is not the case with b_u585i_iot02a board. So a sfdp-bfp property (which will ignore the nor octo-flash one) is useless there. 
In general, better rely on the nor octoflash actual data instead of a DTS composed property.